### PR TITLE
update rekey join tutorial inline with changes to ksqlDB master

### PR DESCRIPTION
**Pull request by big-andy-coates**
This is a larger change than the other updates as the recent changes to key handling made the existing tutorial not really work. It was relying on the fact that a stream without a key column has `ROWKEY` set to `null` if you issue a push query.

Recent changes mean that a stream without an explicit key column have not key column, i.e. no `ROWKEY`, so I've updated the tutorial to use `DESCRIBE` to see the schema of the data and ``PRINT` to view the raw data in Kafka.

Note, needs https://github.com/confluentinc/ksql/pull/5552 to pass.

### CODE REVIEW

File: _includes/tutorials/rekeying/ksql/markup/dev/continuous-rekey.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Let's fix that. Using KSQL's appropriately named `PARTITION BY` clause we can apply a key to the messages and write it to a new stream. Here we'll use the movie identifier, `ID`. Issue the following to create a new stream that is continuously populated by its query:
```
Comment (rmoff) [inline _includes/tutorials/rekeying/ksql/markup/dev/continuous-rekey.adoc:1]: s/KSQL/ksqlDB

**Comment by rmoff** [comment]
Thanks Andy. What version is this going to drop in?

**Comment by big-andy-coates** [comment]
> Thanks Andy. What version is this going to drop in?

0.10 / 6.0

File: _includes/tutorials/rekeying/ksql/markup/dev/continuous-rekey.adoc (hunk 1 above)
Comment (big-andy-coates) [reply _includes/tutorials/rekeying/ksql/markup/dev/continuous-rekey.adoc:1]: This text is deleted ;)

**Pull request merged**